src/applications/form-renderer/utils/formConfig.js

```diff
@@ -0,0 +1,78 @@
+import {
+  dateOfBirthSchema,
+  dateOfBirthUI,
+  fullNameSchema,
+  fullNameUI,
+  titleUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import manifest from '../manifest.json';
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+const selectSchemas = ({ pageTitle, additionalFields }) => {
+  const schemas = {
+    schema: {
+      type: 'object',
+      properties: {
+        fullName: fullNameSchema,
+      },
+    },
+    uiSchema: {
+      ...titleUI(pageTitle),
+      fullName: fullNameUI(),
+    },
+  };
+
+  if (additionalFields.includeDateOfBirth) {
+    schemas.schema.properties.dateOfBirth = dateOfBirthSchema;
+    schemas.uiSchema.dateOfBirth = dateOfBirthUI();
+  }
+
+  return schemas;
+};
+
+const formatChapters = chapters => {
+  const formattedChapters = {};
+
+  chapters.forEach(chapter => {
+    const pages = {};
+    pages[chapter.id] = {
+      path: chapter.id.toString(),
```

Comment:
**MUSING**
I wonder if we're going to want to generate a more human-readable path from a title field of some sort. Having a random ID in the url feels a bit odd. This can be addressed later, certainly.

Reply:
That makes sense. The only issue with using the chapter title is that we cannot currently guarantee that every chapter title within a form will be unique. If we can guarantee that at the CMS level, I would feel comfortable using it instead of the chapter ID.